detekt-formatting/src/main/kotlin/io/gitlab/arturbosch/detekt/formatting/wrappers/PackageName.kt

```diff
@@ -7,7 +7,7 @@ import io.gitlab.arturbosch.detekt.api.internal.AutoCorrectable
 import io.gitlab.arturbosch.detekt.formatting.FormattingRule
 
 /**
- * See [ktlint docs](https://pinterest.github.io/ktlint/rules/standard/#no-underscores-in-package-names) for
+ * See [ktlint docs](https://pinterest.github.io/ktlint/<ktlintVersion/>/rules/standard/#no-underscores-in-package-names) for
```

Comment:
https://pinterest.github.io/ktlint/0.47.0/rules/standard/#no-underscores-in-package-names
https://pinterest.github.io/ktlint/0.48.0/rules/standard/#package-name
https://pinterest.github.io/ktlint/0.49.0/rules/standard/#package-name
https://pinterest.github.io/ktlint/0.50.0/rules/standard/#package-name
```suggestion
 * See [ktlint docs](https://pinterest.github.io/ktlint/<ktlintVersion/>/rules/standard/#package-name) for
```